deploy/olm-catalog/apicast-operator/apicast-operator.v0.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,166 @@
+spec:
+  version: 0.0.0
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: APIcast
+      name: apicasts.apps.3scale.net
+      version: v1alpha1
+      description: APIcast is an API gateway
+      displayName: APIcast
+  description: |
+    The APIcast Operator creates and maintains the Red Hat 3scale API Gateway in several deployment configurations.
+
+    APIcast is an API gateway built on top of [NGINX](https://www.nginx.com/). It is part of the [Red Hat 3scale API Management Platform](https://www.redhat.com/en/technologies/jboss-middleware/3scale).
+
+    ### Supported Features
+    * **Installer** A way to install a APIcast gateway.on
+
+    ### Upgrading your installation
+    The APIcast Operator understands how to run and upgrade between a set of APIcast versions.
+    See [the upgrade guide](https://github.com/3scale/apicast-operator) for more information.
+
+    ### Documentation
+    Documentation can be found on our [website](https://github.com/3scale/apicast-operator).
+
+    ### Getting help
+    If you encounter any issues while using operator, you can create an issue on our [website](https://github.com/3scale/apicast-operator) for bugs, enhancements, or other requests.
+
+    ### Contributing
+    You can contribute by:
+
+    * Raising any issues you find using APIcast Operator
+    * Fixing issues by opening [Pull Requests](https://github.com/3scale/apicast-operator/pulls)
+    * Improving [documentation](https://github.com/3scale/apicast-operator)
+    * Talking about APIcast Operator
+
+    All bugs, tasks or enhancements are tracked as [GitHub issues](https://github.com/3scale/apicast-operator/issues).
+
+    ### License
+    APIcast Operator is licensed under the [Apache 2.0 license](https://github.com/3scale/apicast-operator/blob/master/LICENSE)
+
+  keywords:
+    - 3scale
+    - API
+  displayName: APIcast (nightly)
+  install:
+    spec:
+      deployments:
+      - name: apicast-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: apicast-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: apicast-operator
+            spec:
+              containers:
+              - command:
+                - apicast-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.namespace
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: apicast-operator
+                image: quay.io/3scale/apicast-operator:master
+                imagePullPolicy: Always
+                name: apicast-operator
+                resources: {}
+              serviceAccountName: apicast-operator
+      permissions:
```

Comment:
Before merging this PR, if possible lets wait until https://github.com/3scale/apicast-operator/pull/20 is merged and then regenerate the CSV file. This way we won't have to open another new PR to update it for that.

Reply:
done